docs/Getting Started/Integration Basics/index.md

```diff
@@ -811,7 +823,43 @@ The `sdlManager` must be shutdown properly if this class is shutting down in the
 !@
 
 @![android,javaSE,javaEE]
-### Determining SDL Support
+#### Optional SdlManager Builder Parameters
+!@
+
+@![android,javaSE,javaEE]
+##### App Icon
+This is a custom icon for your application. Please refer to [Adaptive Interface Capabilities](Displaying a User Interface/Adaptive Interface Capabilities) for icon sizes.
+
+```java
+builder.setAppIcon(appIcon);
+```
+
+##### App Type 
+The app type is used by car manufacturers to decide how to categorize your app. Each car manufacturer has a different categorization system. For example, if you set your app type as media, your app will also show up in the audio tab as well as the apps tab of Ford’s SYNC3 head unit. The app type options are: default, communication, media (i.e. music/podcasts/radio), messaging, navigation, projection, information, and social.
+
+```java
+Vector<AppHMIType> appHMITypes = new Vector<>();
+appHMITypes.add(AppHMIType.MEDIA);
+appHMITypes.add(AppHMIType.PROJECTION);
+
+builder.setAppTypes(appHMITypes);
+```
+
+!!! NOTE
+Navigation and projection applications both use video and audio byte streaming. However, navigation apps require special permissions from OEMs, and projection apps are only for internal use by OEMs.
+!!!
```

Comment:
@bilal-alsharifi Are navigation apps available to Java EE / SE apps?

Reply:
Oh no. They are only supported on Android. good catch.